client/my-sites/plans/main.jsx

```diff
@@ -163,12 +157,11 @@ var Plans = React.createClass( {
 						<UpgradesNavigation
 							path={ this.props.context.path }
 							cart={ this.props.cart }
-							selectedSite={ this.props.sites.getSelectedSite() } />
+							selectedSite={ selectedSite } />
 
 						{ this.renderTrialCopy() }
 
 						<PlanList
-							sites={ this.props.sites }
```

Comment:
I think you forgot to pass `site` in here instead. This is probably why I'm not seeing `PlanActions` buttons anywhere:

![screen shot 2016-03-01 at 4 00 27 pm](https://cloud.githubusercontent.com/assets/1130674/13446352/ac1dc7bc-dfc7-11e5-92a8-f3c7751bd098.png)